contents/customers/mintlify.md

```diff
@@ -0,0 +1,59 @@
+    - API
+---
+
+Mintlify is a docs startup used by teams at [Y Combinator](/customers/ycombinator), Explo and  MindsDB to power their product and technical documentation. It helps teams with rapidly evolving products to create rich, well structured docs which convert users and empower colleagues. 
+
+Mintlify has used PostHog internally since launch, making regular use of both the product analytics and session recording tools to help it build better products. 
+
+“We use the product analytics tools, such as funnels, a lot,” says Han Wang, Mintlify’s CEO and Founder. “We also use session recordings too. I actually had to use it last night, to investigate errors on a particular setup. We’ve found recordings particularly useful for support issues like that, but we’re constantly tracking things… Is this CTA engaging? How popular is this page? We monitor everything in PostHog to help us decide where to focus.”
+
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/han.png"
+        size="md"
+        name="Han Wang"
+        title="Founder & CEO, Mintlify"
+        quote={`“You can quote me on this: PostHog is awesome. It’s a great tool. I’ve used a bunch of different analytics platforms in the past and PostHog stands out for it’s developer friendliness and user experience. I really, really love it.”`}
+    />
+</BorderWrapper>
+
+## Session recording, analytics and user feedback
+
+After seeing first-hand how analytics could help them build better products, Mintlify decided to explore ways to give users this same ability within their own product. Rather than building the feature from scratch, the team identified a simpler solution: use PostHog. 
+
+“When we first scoped out user analytics, I was like: ‘Oh, God, do I have to explore some crappy third-vendor solution that does this specifically?’” says Han. “That would've been such a nightmare. Fortunately, we discovered we could just do this with PostHog.”
+
+The initial goal for Mintlify’s analytics was just to give users the ability to information they needed to identify success — basically, what the most popular pages and funnels are. However by talking to users the team discovered that users had needs beyond just this. Mintlify mostly used analytics and funnel insights, but users were also interested in using session recording to track detailed usage. As a result, Mintlify decided to share session recordings with users too. 
+
+Adding session recording massively increased the scope, but just three days later Mintlify had shipped an initial version which included both analytics _and_ session recordings.
+
+## User-facing analytics, powered by PostHog
+
+“We were able to ship this so fast because we had some secret weapons up our sleeve,” says Han. “First was PostHog, which could handle all the data for us and meant there was nothing further to build or configure. Second was a tool called [Tremor](https://www.tremor.so/), which let us just pipe data in and generate some visuals.”
```

Comment:
```suggestion
“We were able to ship this so fast because we had some secret weapons up our sleeve,” says Han. “First was PostHog, which could handle all the data for us and meant there was nothing further to build or configure. Second, was a tool called [Tremor](https://www.tremor.so/), which let us just pipe data in and generate some visuals.”
```